src/pages/Loadouts/Game/[key].js

```diff
@@ -0,0 +1,60 @@
+const games = [
```

Comment:
Dynamic pages for each games
Data would be fetched from the dB, static stub eg for now